test/src/cli/git_cli_test.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:test/test.dart';
+import 'package:very_good_cli/src/cli/cli.dart';
+
+void main() {
+  group('Git', () {
+    group('reachable', () {
+      test('completes for a reachable remote', () async {
+        await expectLater(
+          Git.reachable(
```

Comment:
This test would fail if you don't have GIT on your system or in the CI right? While that would be strange to not have, it would fail incorrectly cause the test itself would probably be fine.

Should we maybe mock the _Cmd somehow?

Reply:
Yeah I was planning to create a separate issue to refactor all CLI classes to support an injectable Process rather than exposing static methods. I'll update this handle checking if git is not installed, good point 👍

Reply:
True but technically that could be skipped by the user, but I am fine with improving it for the tests on a separate PR!

Reply:
https://github.com/VeryGoodOpenSource/very_good_cli/issues/380